contracts/resolvers/PublicResolver.sol

```diff
@@ -51,9 +73,15 @@ contract PublicResolver is ABIResolver, AddrResolver, ContentHashResolver, DNSRe
         emit ApprovalForAll(msg.sender, operator, approved);
     }
 
-    function isAuthorised(bytes32 node) internal override view returns(bool) {
+    function isAuthorised(bytes32 node) internal view override returns (bool) {
+        if (
```

Comment:
Can you point me to which tests cover these logics? I couldn't find one

Reply:
Added one

Reply:
```
it('trusted contract can bypass authorisation', async () => {
        await resolver.methods['setAddr(bytes32,address)'](node, accounts[9], {
          from: accounts[9],
        })
        assert.equal(await resolver.addr(node), accounts[9])
      })
```